src/components/idleMembers/card/index.tsx

```diff
@@ -1,17 +1,22 @@
-import { FC } from 'react';
+import { FC, SyntheticEvent } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
+import { DUMMY_PROFILE } from '@/components/constants/display-sections.js';
+
+const imgUrl = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL;
 
 type Props = {
   idleMemberUserName: string
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
-  const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
-
+  const assigneeProfilePic = (name: string) => `${imgUrl}/${name}/img.png`;
   const getMemberDetails = (name: string) => {
     const newWindow = window.open(`https://members.realdevsquad.com/${name}`, '_blank', ' noopener ,norefferrer');
     if (newWindow) newWindow.opener = null;
   };
+  const assigneeImageOnError = (e: SyntheticEvent<HTMLImageElement>) => {
+    e.currentTarget.src = DUMMY_PROFILE;
```

Comment:
Are you sure this works? Can you please provide a screenshot of the frontend where this dummy image is being used?

Reply:
![image](https://user-images.githubusercontent.com/86315672/132952042-608eabf0-d006-4c5b-9fe8-abd43e0d75ac.png)
Here it is, when I purposely create an error this is the frontend I get

Reply:
And if I use `e.target.src = DUMMY_PROFILE;` then I am getting an error - Property 'src' does not exist on type 'EventTarget'

Reply:
Great. Thanks for the clarification! Looks good then! Also could you add screenshot of properly working images?

Reply:
![image](https://user-images.githubusercontent.com/86315672/132954190-89761c7a-a96e-40f3-b589-17db80e40e1f.

Reply:
Cool!